doc/introduction/interfaces/numpy.rst

```diff
@@ -326,3 +326,54 @@ of supported and unsupported features. We highlight a few of the major 'gotchas'
 ..
 
 - Some ``isinstance`` checks, like ``isinstance(x, np.ndarray)`` or ``isinstance(x, tuple)``, without first doing ``from autograd.builtins import isinstance, tuple``.
+
+
+SciPy Optimization
+------------------
+
+In addition to using autodifferentiation provided by Autograd, the NumPy interface
+also allows QNodes to be optimized directly using the
+`SciPy optimize <https://docs.scipy.org/doc/scipy/reference/optimize.html>`__ module.
+
+Simply pass the QNode, or your hybrid cost function containing QNodes, directly
+to the ``scipy.minimize`` function:
+
+.. code-block:: python
+
+    from scipy.optimize import minimize
+
+    dev = qml.device('default.qubit', wires=2)
+
+    @qml.qnode(dev)
+    def circuit(x):
+        qml.RX(x[0], wires=0)
+        qml.RZ(x[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+        qml.RX(x[2], wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+    def cost(x):
+        return np.abs(circuit(x) - 0.5) ** 2
+
+    params = np.array([0.011, 0.012, 0.05])
+
+    minimize(cost, params, method='BFGS')
+
+Some of the SciPy minimization methods require information about the gradient
+of the cost function via the ``jac`` keyword argument. This is easy to include; we
+can simply create a function that computes the gradient using ``qml.grad``:
+
+>>> minimize(cost, params, method='BFGS', jac=lambda x: qml.grad(cost)(x)[0])
```

Comment:
The need to wrap the gradient function with `lambda` and index into element 0 will be unnecessary once #1081 is merged, and can simply be written `jac=qml.grad(cost)`.

Reply:
Huge relief that change!

Reply:
Nice addition!